frontend/internal-packages/agent/src/checkpoint/SupabaseCheckpointSaver.test.ts

```diff
@@ -636,12 +639,12 @@ describe('SupabaseCheckpointSaver', () => {
       await saver.put(config, checkpoint, metadata, checkpoint.channel_versions)
 
       // Assert
-      if (
-        capturedSaveData &&
-        typeof capturedSaveData === 'object' &&
-        'organization_id' in capturedSaveData
-      ) {
-        expect(capturedSaveData.organization_id).toBe(TEST_ORG_ID)
+      if (capturedSaveData && typeof capturedSaveData === 'object') {
+        // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+        const data = capturedSaveData as {
+          p_checkpoint: { organization_id: string }
+        }
+        expect(data.p_checkpoint.organization_id).toBe(TEST_ORG_ID)
```

Comment:
Since we are verifying this in a test, we have decided to allow type casting here.